ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -337,14 +337,14 @@ pub enum CContextMenuResult {
     Selected,
     // Can't use Dismissed. Already used by PromptResult. See:
     // https://github.com/servo/servo/issues/25986
-    Dismissed_,
+    Dismiss,
```

Comment:
will this break the C header?

Reply:
Break meaning change it? - yes:
```9c9
<   Dismissed_,
---
>   Dismiss,
```
I suppose this might be problem as some other projects probably use this header

Reply:
Hmm… I didn't think this through, we use the C header in our Java binding… Would there be a way to use `Dismissed_` in the C header and `Dismissed` in the C++ header?

Reply:
Break as in make it impossible to use because of the clash